lib/jxl/dec_frame.cc

```diff
@@ -699,6 +697,13 @@ void FrameDecoder::PreparePipeline() {
   if ((frame_header_.flags & FrameHeader::kSplines) != 0) {
     JXL_ABORT("Not implemented: splines");
   }
+  if (frame_header_.upsampling != 1) {
```

Comment:
I know extra channels aren't implemented yet anyway in the render pipeline, but it probably wouldn't hurt to add a TODO here to remember also looking at the extra channel upsampling...